generators/client/templates/angular/package.json.ejs

```diff
@@ -164,6 +164,7 @@
     "postinstall": "webdriver-manager update --gecko false",
     <%_ } _%>
     "start": "<%= clientPackageManager %> run webpack:dev",
+    "start-tls": "<%= clientPackageManager %> run  start --env.tls",
```

Comment:
This isn't passing `--env.tls` to webpack, you might need to add a `webpack:dev-tls` script or figure out how to pass the flag through the chain of npm scripts.
```
> mono@0.0.0 start-tls /private/tmp/jh/mono2
> npm run  start --env.tls


> mono@0.0.0 start /private/tmp/jh/mono2
> npm run webpack:dev
```

Reply:
Also there is an extra space before start

Reply:
Another solution (although a bit ugly) could be to add `-- -- ` to go through the 2 commands:

"start-tls": "<%= clientPackageManager %> run start -- -- --env.tls",